compiler/parser/parser.cpp

```diff
@@ -662,14 +667,17 @@ static ModuleSymbol* parseFile(const char* path,
 
       if (moduleCount == 1) {
         retval = moduleLast;
+      } else if (include) {
+        USR_FATAL(moduleLast, "included module file contains multiple modules");
```

Comment:
This error doesn't seem as though it has a test exercising it / locking it in?  (based on searching for the string on in the github file list)

Reply:
added test IncludeMultipleModules